src/Illuminate/Database/Schema/PostgresBuilder.php

```diff
@@ -16,6 +16,8 @@ public function hasTable($table)
             $schema = head($schema);
         }
 
+        $schema ? $schema : 'public';
```

Comment:
This doesn't actually do anything does it? You aren't assigning the result to a variable